test/api/activity_types_api_test.rb

```diff
@@ -50,4 +50,42 @@ def test_put_activity_types
     first_activity_type = ActivityType.first
     assert_json_matches_model(last_response_body, first_activity_type, response_keys)
   end
+
+  def test_delete_activity_type
+    # Create a activity type
+    activity_type = FactoryBot.create(:activity_type)
+    id_activity_type= activity_type.id
+    # number of activity type before delete
+    number_of_ativity_type = ActivityType.count
+    
+    # perform the delete
+    delete_json with_auth_token "/api/activity_types/#{activity_type.id}"
+    
+    # Check if the delete get through
+    assert_equal 200, last_response.status
+
+    # Check delete if success
+    assert_equal ActivityType.count, number_of_ativity_type-1
+
+    # Check that you can't find the deleted id
+    refute ActivityType.exists?(id_activity_type)
+  end
+
+
+  def test_delete_activity_type_not_auth
+
+     # A user with student role which does not have permision to delete a activity type
+    user = FactoryBot.build(:user, :student)
+        
+     # create a activity type to delete
+    activity_type = FactoryBot.create(:activity_type)
+
+    # perform the delete
+    delete_json with_auth_token("/api/activity_types/#{activity_type.id}", user)
+    
+    # check if the delete does not get through
+    assert_equal 403, last_response.status
+
```

Comment:
Please add another check to see you still can find the activity type you attempted to delete and another to see the number of activity type is still the same